### PR TITLE
chore: Port tests to GHA

### PR DESCRIPTION
**Pull request by cafuego**
* Port all tests from Travis to GHA
* Ensure a docker build runs during testing
* Use vars not secrets for slack channel, so we can *see* what it is

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+            <code>${{ steps.lint.outputs.stdout }}</code>
+            <code>${{ steps.lint.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>PHPCS</summary>
+            <code>${{ steps.phpcs.outputs.stdout }}</code>
+            <code>${{ steps.phpcs.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>Tests</summary>
```
Comment (lazysoundsystem) [inline .github/workflows/run-tests.yml:155]: Doesn't make much difference, but we could skip this and the Site Install if we aren't running those steps.

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+            <code>${{ steps.tests.outputs.stdout }}</code>
+            <code>${{ steps.tests.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>Drupal Logs</summary>
+            <code>${{ steps.dblog.outputs.stdout }} </code>
+            <code>${{ steps.dblog.outputs.stderr }} </code>
+            </details>
+
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:164]:
```suggestion
```

Comment (cafuego) [reply .github/workflows/run-tests.yml:164]: Oh, I thought I'd removed these already. Indeed. Begone!

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ### Test Run Output
+            #### Software Versions
+            #### Composer Validate `${{ steps.validate.outcome }}`
+            #### PHP Lint `${{ steps.lint.outcome }}`
+            #### Docker Build `${{ steps.build.outcome }}`
+            #### Environment Setup `${{ steps.docker.outcome }}`
+            #### Site Install `${{ steps.install.outcome }}`
```
Comment (cafuego) [inline .github/workflows/run-tests.yml:132]:
```suggestion
```

**Pull request merged**